apis/v1alpha1/opampbridge_webhook.go

```diff
@@ -15,33 +15,69 @@
 package v1alpha1
 
 import (
+	"context"
 	"fmt"
 	"strings"
 
+	"github.com/go-logr/logr"
+
+	"github.com/open-telemetry/opentelemetry-operator/internal/config"
+
```

Comment:
the structure of these imports is wrong, though idk why our linter isn't catching it...

Reply:
im going to leave this in for now, and then we can figure out why the linter isn't catching this in a follow up

Reply:
> the structure of these imports is wrong, though idk why our linter isn't catching it...

Yeah. Sometimes, it throws error for correct import structure too.